lib/replay_api/configuration.rb

```diff
@@ -11,12 +11,14 @@ def self.configure
   class Configuration
     attr_accessor :replay_key
     attr_accessor :replay_server
+    attr_accessor :replay_library
     attr_accessor :ssl
     attr_accessor :max_request_batch_size
     attr_accessor :max_request_batch_delay
 
     def initialize
       @replay_server = 'api.replay.io'
+      @replay_library = "Replay Ruby #{VERSION}"
```

Comment:
as it is `ReplayApi` version is directly referrerd to https://github.com/Originate/replay-ruby/blob/send-library-version-to-server/lib/replay_api/version.rb